ginivision/src/main/res/layout/gv_item_thumbnail.xml

```diff
@@ -18,19 +18,34 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+        <ProgressBar
+            android:id="@+id/gv_activity_indicator"
+            android:layout_width="64dp"
+            android:layout_height="match_parent"
+            android:layout_alignParentTop="true"
+            android:layout_centerHorizontal="true"
+            android:layout_above="@id/gv_handle"
+            android:indeterminateOnly="true"
+            android:indeterminateTint="@color/gv_analysis_activity_indicator"
+            android:indeterminateTintMode="src_in"
+            android:padding="@dimen/gv_review_progressbar_padding"
+            android:visibility="invisible"
+            tools:targetApi="lollipop"
```

Comment:
Could this be a problem for preLollipop APIs?

Reply:
No, the progress indicator circle cannot be tinted pre-Lollipop and so tinting will be ignored, which is fine.